libr/core/disasm.c

```diff
@@ -3773,7 +3773,8 @@ static void ds_print_relocs(RDisasmState *ds) {
 		if (rel->import) {
 			r_cons_printf ("; RELOC %d %s", rel->type, rel->import->name);
 		} else if (rel->symbol) {
-			r_cons_printf ("; RELOC %d %s", rel->type, rel->symbol->name);
+			r_cons_printf ("; RELOC %d %s @ 0x%llx + 0x%llx", rel->type,
```

Comment:
use PFMT64x, llx is not portable, also pad with %08

Reply:
one of the offsets is duplicated

Reply:
what do you mean by duplicated? where?

Reply:
i will merge if you use PFMT64x instead of llx, pls update